README.md

```diff
@@ -121,15 +120,17 @@ Or to run an individual test file:
 ./vendor/bin/phpunit tests/MagicTest.php
 ```
 
-The library uses [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) for code formatting. Code must be formatted before PRs are submitted. Run the formatter with:
+The library uses [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) for code formatting.
+Code must be formatted before PRs are submitted. Run the formatter with:
 
 ```bash
-php-cs-fixer fix -v --using-cache=no .
+./vendor/bin/php-cs-fixer fix -v --using-cache=no .
```

Comment:
I see!! `friendsofphp/php-cs-fixer` is not required.

Could we add it? Or mention that this tool must be installed globally and must on `$PATH`.

Reply:
Sure, I think global installation instructions would be an improvement. Perhaps something like this from [here](https://cs.symfony.com/doc/installation.html)?

```
composer global require friendsofphp/php-cs-fixer
export PATH="$PATH:$HOME/.composer/vendor/bin"
```

Reply:
BTW I will tag a new release tomorrow.

Reply:
Linked. 🍏